doc/faq.rst

```diff
@@ -199,6 +199,23 @@ If Binder sessions take a while to start, but you know that your image has
 already been built, there's a good chance you are in step 2, and the server is
 still pulling the image onto the node that you'll be using. Please be patient!
 
+Will repos with fewer notebooks launch faster? Should I split my notebooks into smaller repos?
+----------------------------------------------------------------------------------------------
+
+Number of notebooks in a repo shouldn't have any impact on binder launch time,
+assuming a docker image for that repo is already built. It is worth noting, however,
+that there is a limit to the number of repos that can be active at any moment in 
```

Comment:
maybe switch "to the number of repos" -> "to the number of instances of a repository"? Just to clarify that you can have as many repositories as you like, but each one will only be allowed a certain number of active instances on mybinder.org